src/tests/dictionaryConcepts/components/DescriptionTable.test.jsx

```diff
@@ -30,4 +30,20 @@ describe('Test suite for DescriptionTable', () => {
     const wrapper = shallow(<DescriptionTable {...props} />);
     expect(wrapper).toMatchSnapshot();
   });
+  it('should render DescriptionTable Component for new concepts', () => {
+    const newProps = {
+      ...props,
+      existingConcept: {},
+      description: [{
+        uuid: 'newUUID',
+      }, {
+        uuid: 'a.n.other',
+      }],
+    };
+    const wrapper = mount(<DescriptionTable {...newProps} />);
+    expect(wrapper.find('DescriptionRow').length).toEqual(newProps.description.length);
+    expect(wrapper.find('DescriptionRow').at(0).instance().props.newRow)
+      .toEqual(newProps.description[0]);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Given we are updating snapshots each time while testing, is snapshot testing still necessary?

Reply:
No, we are not updating snapshots, locally. Check on the tests for other components.